setup.py

```diff
@@ -19,7 +19,7 @@ def read(*parts):
     os.system('python setup.py sdist upload')
     os.system('python setup.py bdist_wheel upload')
     print('You probably want to also tag the version now:')
-    print('  git tag -a %s -m "version %s"' % (version, version))
+    print(('  git tag -a %s -m "version %s"' % (version, version)))
```

Comment:
This change shouldn't be made as a single brace should suffice for this task.

Reply:
reverted.